jobfunnel/backend/scrapers/base.py

```diff
@@ -351,18 +356,6 @@ def scrape_job(self, job_soup: BeautifulSoup, delay: float,
         return job
     # pylint: enable=no-member
 
-    @abstractmethod
-    def get_job_soups_from_search_result_listings(self) -> List[BeautifulSoup]:
-        """Scrapes a job provider's response to a search query where we are
-        shown many job listings at once.
-
-        NOTE: the soups list returned by this method should contain enough
-        information to set your self.min_required_job_fields with get()
-
-        Returns:
-            List[BeautifulSoup]: list of jobs soups we can use to make a Job
-        """
-
```

Comment:
while I am in favour of implementing abstract classes to get job pages -> listings -> soups, I think we should put this workflow into it's own class such as `BaseMultiPageScraper` so that we can write scrapers for static web-pages that only have a single page (sort of like monster). 

I think this way we can do the single-page scroll type of job sites as a `BaseSinglePageScraper`.